explainer.md

```diff
@@ -1,3 +1,118 @@
 ## Explainer
 
-### Examples
+### New data fields in RTCRtpContributingSource for audio-video synchronization and end-to-end delay measurements
+
+Authors:
+minyue@chromium.org
+hbos@chromium.org
+chxg@google.com
+
+Participate
+https://www.chromestatus.com/feature/5728533701722112
+
+#### Abstract
+
+This document provides a solution for real-time communication systems measure end-to-end delay and audio video synchronization performance. This solution is particularly desired by systems, where an intermediate steam regenerator that terminates the streams originating from senders, an audiuo mixer as an example, is involved.
+
+#### Introduction
+
+In simple real-time communication systems, which involves only a sender and a receiver, the RTCP scheme [RFC3550] allows estimation of round-trip time and thus end-to-end delay.
+However, in a more sophisticated system, where an intermediate stream regenerator is involved, the estimation of end-to-end delay becomes more difficult. For example, if audio mixing is performed on the server side, the server terminates inbound media streams, processes the media data, and then generates new outbound media streams with fresh synchronization information.
+
+The solution proposed in this document is based on a new RTP header extension, namely absolute capture time, which contains two data fields:
+ * an absolute capture timestamp, which is stamped by the original capturer, and is supposed to be received untouched by the end receivers.
+ * an estimated clock offset with respect to the capturer's clock, which is supposed to be read and updated by every intermediate sender.
+
+With the absolute capture timestamps, end receivers can accurately measure the audio-video synchronization performance. With the `estimated clock offset`, which gets updated at each intermediate hop, end receivers can estimate their respective clock offset with respect to the capturer's clock, and then together with the absolute capture timestamp, measure the end-to-end delay. 
+
+The absolute capture time RTP header extension has been adopted by WebRTC, see the specification [here](https://github.com/webrtc/webrtc-org/blob/gh-pages/experiments/rtp-hdrext/abs-capture-time/index.md).
+
+To surface the information to web applications, two data fields are proposed to be added to RTCRtpContributingSource, and has been adopted by WebRTC 1.0 extensions, see the specification [here](https://w3c.github.io/webrtc-extensions/#rtcrtpcontributingsource-dictionary).
```

Comment:
I'd say "The WebRTC Extensions spec". This spec contains post-1.0 features.

However this explainer is on the spec in question's repo, so I would just say "This specification exposes (...the things it exposes...)"